elide-async/src/main/java/com/yahoo/elide/async/models/AsyncQuery.java

```diff
@@ -35,18 +38,23 @@
 public class AsyncQuery extends AsyncBase implements PrincipalOwned {
     @Id
     @Column(columnDefinition = "varchar(36)")
-    @Pattern(regexp = "^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$",
-            message = "id not of pattern UUID")
-    private String id; //Provided.
+    private String id; //Provided by client or generated if missing on create.
```

Comment:
`@Pattern` got removed due to merge, we should add it back.

Reply:
I removed it because then it gives me an error when id is not passed at create time.

Reply:
if it throws error when its null or empty, then please go ahead and remove it.